extensions/Push/lib/Connector/Webhook.pm

```diff
@@ -84,6 +84,32 @@ sub should_send {
     }
   }
 
+  # check if the bug was removed from a product/component we care about
+  if ($event =~ /change/ && $message->routing_key =~ /\Qbug.modify\E/) {
+    my $removed_product = "";
+    my $removed_component = "";
+    if (exists $bug_data->{'changes'}) {
+      foreach my $change ($bug_data->{'changes'}) {
+        if ($change->{'field'} eq 'product') {
+          $removed_product = $change->{'removed'};
+        }
+        if ($change->{'field'} eq 'component') {
+          $removed_component = $change->{'removed'};
+        }
+      }
+    }
+    if ($removed_product || $removed_component) {
+      if ($removed_product eq '') {
+        $removed_product eq $bug_data->{'product'};
```

Comment:
$removed_product = $bug_data->{'product'};

Reply:
Bwahahaha, I can't believe I did that.

Reply:
Ok, fixed.